snap.svg.zpd.js

```diff
@@ -423,7 +440,7 @@ SVGElement.prototype.getTransformToElement = SVGElement.prototype.getTransformTo
                     // Pan mode
                     var p = _getEventPoint(event, zpdElement.data.svg).matrixTransform(zpdElement.data.stateTf);
 
-                    _setCTM(g, zpdElement.data.stateTf.inverse().translate(p.x - zpdElement.data.stateOrigin.x, p.y - zpdElement.data.stateOrigin.y), zpdElement.options.zoomThreshold);
+                    _setCTM(g, zpdElement.data.stateTf.inverse().translate(p.x - zpdElement.data.stateOrigin.x, p.y - zpdElement.data.stateOrigin.y));
```

Comment:
If you don't mind, I would like to leave the `threshold` here as the parameter so the function `_setCTM` doesn't need to care about what's the options.

hope this is fine.

overall great 👍